runtime/oti/j9javaaccessflags.h

```diff
@@ -30,112 +30,132 @@
 
 /* @ddr_namespace: map_to_type=J9JavaAccessFlags */
 
-/* Constants from J9JavaAccessFlags */
+/* ROM class modifiers */
+#define J9AccPublic 0x1
+#define J9AccPrivate 0x2
+#define J9AccProtected 0x4
+#define J9AccStatic 0x8
+#define J9AccFinal 0x10
+/* Project Valhalla repurposes 0x20 as ACC_IDENTITY since ACC_SUPER has no effect after Java 8. */
+#define J9AccSuper 0x20
+#define J9AccClassHasIdentity 0x20
+#define J9AccValueType 0x40
+#define J9AccInterface 0x200
 #define J9AccAbstract 0x400
+#define J9AccPrimitiveValueType 0x800
 #define J9AccAnnotation 0x2000
-#define J9AccBridge 0x40
+#define J9AccEnum 0x4000
+#define J9AccClassArray 0x10000
+#define J9AccClassInternalPrimitiveType 0x20000
+/* Masks and helpers */
+#define J9AccClassCompatibilityMask 0x7FFF
 
-#define J9AccClassHasIdentity 0x20
+/* ROM class extra modifiers.
+ * See map in ROMClassBuilder::computeExtraModifiers for
+ * available slots.
+ */
 #define J9AccImplicitCreateHasDefaultValue 0x10
 #define J9AccImplicitCreateNonAtomic 0x20
 #define J9AccClassIsValueBased 0x40
 #define J9AccClassHiddenOptionNestmate 0x80
 #define J9AccClassHiddenOptionStrong 0x100
-#define J9AccClassAnnnotionRefersDoubleSlotEntry 0x80000
+#define J9AccSealed 0x200
+#define J9AccRecord 0x400
 #define J9AccClassAnonClass 0x800
-#define J9AccClassArray 0x10000
-#define J9AccClassBytecodesModified 0x100000
-#define J9AccClassCloneable 0x80000000
-#define J9AccClassCompatibilityMask 0x7FFF
-#define J9AccClassDepthMask 0xFFFF
-#define J9AccClassDying 0x8000000
-#define J9AccClassFinalizeNeeded 0x40000000
-#define J9AccClassGCSpecial 0x800000
-#define J9AccClassHasBeenOverridden 0x100000
-#define J9AccClassHasClinit 0x4000000
-#define J9AccClassHasEmptyFinalize 0x200000
-#define J9AccClassIsUnmodifiable 0x400000
-#define J9AccClassHasFinalFields 0x2000000
-#define J9AccClassHasJDBCNatives 0x400000
-#define J9AccClassHasNonStaticNonAbstractMethods 0x8000000
-#define J9AccClassHasVerifyData 0x800000
-#define J9AccClassHotSwappedOut 0x4000000
+#define J9AccSynthetic 0x1000 // TODO why isn't this in modifiers?
```

Comment:
I think it _does_ belong in the modifiers group.

Reply:
Woops I meant to remove that comment. See https://github.com/eclipse-openj9/openj9/issues/17785#issuecomment-1634876543 and let me know what you think.